edit-post/assets/stylesheets/main.scss

```diff
@@ -73,6 +73,11 @@ body.gutenberg-editor-page {
 	ul#adminmenu > li.current > a.current::after {
 		border-right-color: $white;
 	}
+
+	.media-frame select.attachment-filters:last-of-type {
```

Comment:
I wonder if this is a Core Bug and if it should be backported to Core somehow?

Reply:
I initially thought about submitting a ticket to trac for this, but I have never seen the issue outside of Gutenberg. I can test it a little later today to see if this is a core issue.

Reply:
So the question is:

 - What gutenberg styles are conflicting with this? Maybe we can fix it differently in that case.
 - Another possibility is that it's a Core bug surfaced by Gutenberg

Reply:
This is just an issue because core (non-gutenberg media modal) has two dropdowns there, with the other being for filetype. It uses two declarations for them to appear side by side: 
```css
.media-modal-content .media-frame select.attachment-filters {
    margin-top: 11px;
    margin-right: 2%;
    width: 42%;
    width: (48% - 12px);
```
```css
.media-frame select.attachment-filters {
    margin-top: 11px;
    margin-right: 2%;
    max-width: 42%;
    max-width: calc(48% - 12px);
```
In our case, we only have one, so those rules are basically chopping it in half. We need to override those.

Reply:
I think it's a core issue then. Because we're relying on `wp.media` APIs to open the modal, we're not hiding the second select with CSS which means if the `wp.media` allows us to show the media modal in that shape, it should also show it properly.

Reply:
I will submit a ticket on trac for this and reference the original issue #2202

Reply:
Ticket/Patch 
https://core.trac.wordpress.org/ticket/44760